lib/libc/stdlib/malloc/mrs/mrs.c

```diff
@@ -1316,6 +1321,33 @@ mrs_init_impl_locked(void)
 		exit(7);
 	}
 
+	if (!issetugid()) {
+		char *envstr, *end;
+		if ((envstr = getenv(MALLOC_QUARANTINE_DENOMINATOR_ENV)) !=
```

Comment:
BTW we have `secure_getenv()` in libc now, effectively a helper for `if (!issetugid()) { var = getenv(...); ... }`.

Reply:
Thanks, I'll use that in this commit and circle back to use it everywhere separately